### PR TITLE
Added registration endpoint and documentation

### PR DESCRIPTION
**Pull request by usanaphtal112**

### CODE REVIEW

**Review by AbuuIbtisaam — CHANGES_REQUESTED** [review]
@usanaphtal112 I didn't see requirements.txt file but you have installed some packages that was not there. Please consider to update the requirements.txt and push it again. 

Good job.

**Comment by usanaphtal112** [comment]
> @usanaphtal112 I didn't see requirements.txt file but you have installed some packages that was not there. Please consider to update the requirements.txt and push it again.
> 
> Good job.

Done!! You can check once again!

**Review by AbuuIbtisaam — APPROVED** [review]

**Review by AbuuIbtisaam — APPROVED** [review]

File: sahaget_bn/urls.py [hunk 1]
```diff
@@ -16,7 +16,24 @@
...
     path("admin/", admin.site.urls),
+    path("api/v1/users/", include("accounts.urls")),
+    path("api/v1/schema/", SpectacularAPIView.as_view(), name="schema"),
+    path(
+        "api/v1/schema/redoc/",
+        SpectacularRedocView.as_view(url_name="schema"),
+        name="redoc",
+    ),
+    path(
+        "api/v1/schema/swagger-ui/",
```
Comment (leandreAlly) [inline sahaget_bn/urls.py]: if this is the endpoints for swagger docs renew it to /api/v1/api-docs

**Review by leandreAlly — APPROVED** [review]

File: sahaget_bn/urls.py (hunk 1 above)
Comment (leandreAlly) [reply sahaget_bn/urls.py]: @usanaphtal112

Comment (usanaphtal112) [reply sahaget_bn/urls.py]: Resolved!

**Pull request merged**